Stylesheets/Docs/navigation/docs_navigation.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Navigation | MondayBlues</title>
+    <link rel="stylesheet" href="/Stylesheets/Abstract/fontawesome-free-6.0.0-web/css/all.css">
+    <link rel="shortcut icon" href="/Assets/images/logo-modified.png" type="image/x-icon"> 
+    <link rel="stylesheet" href="/Stylesheets/Docs/docs.css">
+    <link rel="stylesheet" href="/Stylesheets/Docs/codeblocks/codeblocks.css">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/11.4.0/styles/base16/railscasts.min.css"
+        rel="stylesheet" />
+</head>
+
+<body>
+    <div class="navigation-page-contents">
+        <nav class="display-align-center display-justify-space-between">
+            <div class="heading display-align-center">
+                <div class="hamburger-nav clickable-image">
```

Comment:
I like how you have created a hamburger menu of your own, instead of using a icon